app/upgrades/types.go

```diff
@@ -22,3 +29,28 @@ type Upgrade struct {
 	// StoreUpgrades adds, renames and deletes KVStores in the state to prepare for a protocol upgrade.
 	StoreUpgrades storetypes.StoreUpgrades
 }
+
+type grantAuthorization struct {
+	grantee       sdk.AccAddress
+	granter       sdk.AccAddress
+	authorization authz.Authorization
+	expiration    *time.Time
+}
+
+func newTestNetGrantAuthorization(msg string) grantAuthorization {
```

Comment:
#PUC on what the goal of this function is

Reply:
Removed it.